app.js

```diff
@@ -6,14 +6,21 @@ import { execWithRetry } from "./lib/util";
 import { deleteFile, downloadPdf, uploadFile } from "./lib/graph-api";
 
 app.post("/files/:id/convert", async (req, res, next) => {
-  const fileId = req.params.id;
-  const file = await getFile(fileId);
-  file.path = file.physicalUri.replace("share://", "/share/");
-
   try {
+    const fileId = req.params.id;
+    const file = await getFile(fileId);
+    file.path = file.physicalUri.replace("share://", "/share/");
+
     const stats = fs.statSync(file.path);
     file.size = stats.size;
     const readStream = fs.createReadStream(file.path);
+    
+    // If this fails this crashes the app if not caught.
+    readStream.on('error', function(err) {
+      console.log(`Error reading readStream: ${err.message}`);
+      // silence service crashing error, throwing here crashes
+      console.trace(err);
+    });
```

Comment:
not sure how often a `createReadStream` fails. I gave it an incorrect path and it crashes the service.
The only way to prevent that was to do this and now throw the error.
Doing this will still result in an error in the try catch, but no more crashing in that case.